src/game/common/system/localfile.cpp

```diff
@@ -69,29 +69,29 @@ bool LocalFile::Open(const char *filename, int mode)
 
     int openmode = O_RDONLY;
 
-    if ((m_openMode & FileMode::CREATE) != 0) {
+    if ((m_access & CREATE) != 0) {
         openmode |= O_CREAT;
     }
 
-    if ((m_openMode & FileMode::TRUNCATE) != 0) {
+    if ((m_access & TRUNCATE) != 0) {
         openmode |= O_TRUNC;
     }
 
-    if ((m_openMode & FileMode::APPEND) != 0) {
+    if ((m_access & APPEND) != 0) {
         openmode |= O_APPEND;
     }
 
-    if ((m_openMode & FileMode::TEXT) != 0) {
+    if ((m_access & TEXT) != 0) {
         openmode |= O_TEXT;
     }
 
-    if ((m_openMode & FileMode::BINARY) != 0) {
+    if ((m_access & BINARY) != 0) {
         openmode |= O_BINARY;
     }
 
-    if ((m_openMode & (FileMode::READ | FileMode::WRITE)) == (FileMode::READ | FileMode::WRITE)) {
+    if ((m_access & (READ | WRITE)) == (READ | WRITE)) {
         openmode |= O_RDWR;
-    } else if ((m_openMode & FileMode::WRITE) != 0) {
+    } else if ((m_access & WRITE) != 0) {
         openmode |= O_WRONLY | O_CREAT;
     }
```

Comment:
LocalFile::Open end of function differs too, close() happens regardless of if handle is valid

Reply:
fixed.